### PR TITLE
Remove anonymous functions.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Part 1 of 3.

Refactor PHP to remove anonymous functions.
## Relevant Technical Choices
The preg_replace_callbacks, I wasn't sure about replacing these, but I thought, might as well. 

## Testing Instructions

---

Fixes #

### CODE REVIEW

File: includes/KSES.php [hunk 1]
```diff
@@ -540,13 +540,22 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 	public function filter_content_save_pre_before_kses( $post_content ) {
 		return (string) preg_replace_callback(
 			'|(?P<before><\w+(?:-\w+)*\s[^>]*?)style=\\\"(?P<styles>[^"]*)\\\"(?P<after>([^>]+?)*>)|', // Extra slashes appear here because $post_content is pre-slashed..
-			static function ( $matches ) {
-				return $matches['before'] . sprintf( ' data-temp-style="%s" ', $matches['styles'] ) . $matches['after'];
-			},
+			[ $this, 'filter_content_save_pre_before_kses_matches' ],
```
Comment (swissspidy) [inline includes/KSES.php]: I think we should keep the anonymous function here.
WP core standards allow anonymous functions for `preg_replace_callback` too.

For testing, we can just test `filter_content_save_pre_before_kses` itself.

Comment (spacedmonkey) [reply includes/KSES.php]: Agreed. I also noted it might be overkill. I will revert.

Comment (spacedmonkey) [reply includes/KSES.php]: Fixed https://github.com/google/web-stories-wp/pull/2593/commits/5d6850084b0c4f84470e3c48977830551d6bf073

File: includes/Media.php [hunk 2]
```diff
@@ -232,44 +232,23 @@ public static function rest_api_init() {
 			'attachment',
 			'media_source',
 			[
+
```
Comment (swissspidy) [inline includes/Media.php:235]:
```suggestion
```

**Review by swissspidy — APPROVED** [review]
LGTM 👍

**Pull request merged**